Annotation/constituent_golden_set.json

```diff
@@ -1507,7 +1741,7 @@
                     }

```

Comment:
predicate should be "pestering Megan about" for the full span

Reply:
I don't think we've been including recipient arguments within the str? I know I haven't and I didnt see you doing this in your past annotations